app/views/RoomInfoView/components/RoomInfoButtons.tsx

```diff
@@ -93,13 +102,18 @@ export const RoomInfoButtons = ({
 				label={i18n.t(isIgnored ? 'Unignore' : 'Ignore')}
 				iconName='ignore'
 				showIcon={!!renderIgnoreUser}
-				danger
 			/>
 			<BaseButton
 				onPress={handleBlockUser}
-				label={i18n.t(`${isBlocked ? 'Unblock' : 'Block'}_user`)}
+				label={i18n.t(`${isBlocked ? 'Unblock' : 'Block'}`)}
 				iconName='ignore'
 				showIcon={!!renderBlockUser}
+			/>
+			<BaseButton
```

Comment:
I could be wrong, but we've been adding more and more buttons here and it's getting messy, because we don't have a proper grid.
Here you can see `Mensagem` is way bigger than `Ligar`.
I can imagine this getting very ugly in some languages.
Can you equalize the amount of space used?

![image](https://github.com/RocketChat/Rocket.Chat.ReactNative/assets/804994/f45aee6a-58a6-4b07-87f3-519a688cd217)

Reply:
![image](https://github.com/RocketChat/Rocket.Chat.ReactNative/assets/804994/24499790-1a04-4b9c-a90f-f67a33a58e9c)
![image](https://github.com/RocketChat/Rocket.Chat.ReactNative/assets/804994/858527c8-b90f-4df8-95e8-e673d6a8cd2a)